Tools/WebKitTestRunner/wpe/EventSenderProxyClientWPE.cpp

```diff
@@ -264,6 +264,29 @@ void EventSenderProxyClientWPE::keyDown(WKStringRef keyRef, double time, WKEvent
 {
     unsigned modifiers = wkEventModifiersToWPE(wkModifiers);
     auto keyval = wpeKeyvalForKeyRef(keyRef, location, modifiers);
+    unsigned downModifiers = modifiers;
+    switch (keyval) {
+    case WPE_KEY_Control_L:
+    case WPE_KEY_Control_R:
+        downModifiers|= WPE_MODIFIER_KEYBOARD_CONTROL;
```

Comment:
Nit: space missing before the `|=` operator:

```suggestion
        downModifiers |= WPE_MODIFIER_KEYBOARD_CONTROL;
```